### DISCUSSION TITLE
Use liquid resize without cli

### DISCUSSION THREAD

**Discussion by ozebe** [ImageMagick/ImageMagick#3504] [Help]
Can you explain how I can use liquid-resize without using the os.system command? I use PythonMagick and I haven't found a way to use this feature, see my code below:

    blob = PythonMagick.Blob()
    imagens = []
    for foto in fotos:
        im = PythonMagick.Image(f'./' + dirpathInput + '/' + foto)
        #apply liquid-resize effect
        print("Effect applyed in: " + foto)
        im.write(blob, "jpg")
        imagens.append(im)

**Comment by snibgo** [comment]
From the documentation https://docs.wand-py.org/en/0.6.6/wand/image.html ,  liquid_rescale(width,height,delta_x=0,rigidity=0) seems likely.

**Reply by ozebe** [reply]
thank you, im gonna use wand instead, you save me bro, thank you
